app/controllers/users/omniauth_callbacks_controller.rb

```diff
@@ -1,6 +1,6 @@
 class Users::OmniauthCallbacksController < ApplicationController
   # TODO: Remove this once we figure out IST, this is only required for the omniauth developer provider
-  skip_before_action :verify_authenticity_token, only: :complete unless Rails.env.production?
+  skip_before_action :verify_authenticity_token, only: :complete unless (Rails.env.production? && !ENV['ENABLE_TMP_LOGINS'])
```

Comment:
Yeah we could override it or something. Code lives here:
https://github.com/omniauth/omniauth/blob/master/lib/omniauth/strategies/developer.rb
https://github.com/omniauth/omniauth/blob/master/lib/omniauth/form.rb

I am hoping this is temporary anyways. Hopefully testshib/IST SAML will replace the need for this code